### ISSUE TITLE
Error in cleanMicrodata.js on nature.com

### ISSUE THREAD

**Issue by garymoon** [MattMcFarland/SUq#9]
Hi Matt,

In putting SUq through its paces, I've had the following error present itself any time I attempt to SUq a page at nature.com. Unlike the EventEmitter issue I filed earlier, I wasn't able to reproduce it locally, and have only reproduced it on Ubuntu 14.04 (I'm using node v5.5.0 but it also happens on v4.4.1).

```
node_modules/suq/lib/cleanMicrodata.js:31
        acc[this.parent.parent.key] = '__ref__' + this.parent.parent.node[0].type[1];
                                                                                 ^

TypeError: Cannot read property '1' of undefined
    at Object.<anonymous> (node_modules/suq/lib/cleanMicrodata.js:31:82)
    at Object.<anonymous> (node_modules/traverse/index.js:59:22)
    at walker (node_modules/traverse/index.js:190:22)
    at node_modules/traverse/index.js:208:29
    at Array.forEach (native)
    at forEach (node_modules/traverse/index.js:298:31)
    at walker (node_modules/traverse/index.js:203:13)
    at node_modules/traverse/index.js:208:29
    at Array.forEach (native)
    at forEach (node_modules/traverse/index.js:298:31)
```

You can attempt to reproduce it using:

```
var suq = require('suq');

var url = "http://www.nature.com/nature/journal/vaop/ncurrent/full/nature16533.html";

suq(url, function (err, json, body) {

    if (!err) {
        console.log('scraped json is:', JSON.stringify(json, null, 2));
        //console.log('html body is', body);
    }

});
```

If it doesn't stand out to you what the issue is let me know and I'll dig a little deeper.

Cheers,
Gary.

**Comment by MattMcFarland** [comment]
It looks like type is undefined, that might be a possibility in some cases. 

Especially this:

``` js
this.parent.parent.node[0].type[1];
```

 So the fix might be to make sure type exists before trying to get [1] of it, as to avoid an exception.

Might be useful doing something like this:

``` js
var node = _.get(this, 'parent.parent.node');

if (node && node[0] && node[0].type && node[0].type[1]) {
}
```

**Comment by garymoon** [comment]
Hi Matt,

Thanks, worked perfectly! I've included it in PR #10, and tested it against a handful of sites in addition to nature.com. Please let me know if you have any concerns.

Cheers,
Gary.
